plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -58,21 +66,31 @@ class NewServiceFormModal extends Component {
       return;
     }
 
+    if (serviceReviewActive) {
+      // Just hide review screen. Form or JSON mode will be
+      // activate dautomaticaly depending on their last state
+      this.setState({
+        serviceReviewActive: false
+      });
+      return;
```

Comment:
We could also just set state at the bottom and let these `if`s determine the updated state